tests/scripts/manage_e2e_tests.sh

```diff
@@ -130,7 +130,7 @@ function run_tests() {
       GINKGO_PARAMS+=("${param}" "${!envName}")
     fi
   done
-  GINKGO_PARAMS+=("${RAW_GINKGO[@]}")
+  GINKGO_PARAMS+=("${RAW_GINKGO[@]:-}")
```

Comment:
[0] I've run into the "RAW_GINKGO unset" error before but was never sure whether it was a bad/unnecessary assumption made by the script, or whether it indicated an actual environmental issue on my part.

I don't understand the GINKGO stuff in general well enough to chime in.  Curious if you had any thoughts @HoustonPutman ?